packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -661,11 +664,68 @@ class HeapTreeViewState extends State<HeapTree>
     selectFromSearchField(controller, foundName);
   }
 
+  Widget trackerWidget(MemoryController controller) {
+    String trackingName;
+    // First class that matches not exact match until editing is completed.
+    ClassHeapDetailStats firstMatchClass;
+
+    final textController = TextEditingController();
+
+    return controller.monitorAllocations.isNotEmpty
+        ? Container(
+            width: defaultSearchTextWidth,
+            height: defaultTextFieldHeight,
+            child: TextField(
+              key: trackAllocationKey,
+              controller: textController,
+              inputFormatters: [
+                // Valid Dart class name.
+                FilteringTextInputFormatter.allow(
+                  RegExp(r'^[a-zA-Z_$][a-zA-Z_$0-9]*$'),
+                )
+              ],
+              onChanged: (text) {
+                firstMatchClass = controller.monitorAllocations.firstWhere(
+                  (element) => element.classRef.name.startsWith(text),
+                  orElse: () => null,
+                );
+                // TODO(terry): Use search auto-complete.
+                trackingName = text;
+              },
+              onEditingComplete: () {
+                if (firstMatchClass == null) {
+                  // TODO(terry): Use search auto-complete.
+                  debugLogger('Class $trackingName not found');
+                } else {
+                  final foundClass = controller.monitorAllocations.firstWhere(
+                    (element) => element.classRef.name == trackingName,
+                    orElse: () => null,
+                  );
+                  foundClass.isStacktraced = true;
+                  controller.setTracking(foundClass.classRef, true);
+                  textController.clear();
```

Comment:
we need to display a toast at least the first time this is done that the results only show up for new allocations.

Reply:
Added TODO.  We have a message bus for toast but I don't see a toast to display when event is received. Do we have a toast in DevTools (I thought we did maybe it never got ported from HTML to Flutter).